dask/dataframe/io/tests/test_json.py

```diff
@@ -120,6 +121,26 @@ def _my_json_reader(*args, **kwargs):
         assert_eq(actual, actual_pd)
 
 
+@pytest.mark.parametrize(
+    "engine",
+    [
+        pytest.param(
+            "ujson",
+            marks=pytest.mark.skipif(
```

Comment:
Could you configure the test to check the error message if pandas < 2.0 instead of skipping it?

Reply:
Yeah, sorry about that!